### PR TITLE
Fix: inotify read timeout not in ms

### PR DESCRIPTION
**Pull request by grembo**
This was off by a factor of 1000, leading to a lot more invocations of the loop body than necessary.

## Proposed change

Calculate the timeout passed to inotify.read to be in milliseconds (as required by the function) instead of seconds.

The issue becomes easily perceivable when setting `PAPERLESS_CONSUMER_INOTIFY_DELAY` to a high number, so that the while loop becomes obvious in tools like top.

## Type of change

- [x] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [ ] Documentation only.
- [ ] Other. Please explain:

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: src/documents/management/commands/document_consumer.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def handle_inotify(self, directory, recursive, is_testing: bool):
                 # If files are waiting, need to exit read() to check them
                 # Otherwise, go back to infinite sleep time, but only if not testing
                 if len(notified_files) > 0:
-                    timeout = inotify_debounce
+                    timeout = inotify_debounce * 1000
```
Comment (grembo) [inline src/documents/management/commands/document_consumer.py:345]: might make sense to rename timeout to timeout_ms to make the code more explicit - just wanted to keep the change as small as possible.

**Comment by shamoon** [comment]
Seems like a good catch to me, thanks. I'll let the person generally most versed in this side of things chime in before merging.

https://github.com/chrisjbillington/inotify_simple/blob/76e8af8705ee4fb3acd926d66a1ba3de0f5d78d5/inotify_simple.py#L124

**Review by shamoon — APPROVED** [review]

**Review by stumpylog — CHANGES_REQUESTED** [review]
Good catch, but lets tweak it a little bit

Lets calculate `inotify_debounce * 1000` only once, and toss a `_ms` so no one forgets this value is in milliseconds

**Comment by grembo** [comment]
@stumpylog Do you mean something like this? Note that we still need a seconds representation of inotify_debounce within the loop body - therefore my solution would have been to rename it to inotify_debounce_secs (like I did), but keep the multiplication * 1000 - this iteration introduces the extra variable like requested (and also renames timeout to timeout_ms to make this explicit).

File: src/documents/management/commands/document_consumer.py [hunk 2]
```diff
@@ -298,15 +298,16 @@ def handle_inotify(self, directory, recursive, is_testing: bool):
         else:
             descriptor = inotify.add_watch(directory, inotify_flags)
 
-        inotify_debounce: Final[float] = settings.CONSUMER_INOTIFY_DELAY
+        inotify_debounce_secs: Final[float] = settings.CONSUMER_INOTIFY_DELAY
```
Comment (shamoon) [inline src/documents/management/commands/document_consumer.py:301]: No reason to have create a  _secs variable imho, just create _ms one with settings.CONSUMER_INOTIFY_DELAY * 1000

Comment (shamoon) [reply src/documents/management/commands/document_consumer.py:301]: Oh scratch this

Comment (grembo) [reply src/documents/management/commands/document_consumer.py:301]: Nope, we need seconds within the function body

```
waited_long_enough = (
                        monotonic() - last_event_time
                    ) > inotify_debounce_secs
```

**Review by stumpylog — APPROVED** [review]
Looks good, thank you

**Pull request merged**